pkg/segment/reader/record/recordreader.go

```diff
@@ -42,11 +42,6 @@ import (
 func GetRecordsFromSegment(segKey string, vTable string, blkRecIndexes map[uint16]map[uint16]uint64,
 	tsKey string, esQuery bool, qid uint64,
 	aggs *structs.QueryAggregators, colsIndexMap map[string]int) (map[string]map[string]interface{}, map[string]bool, error) {
-
-	if aggs.HasGenerateEvent() {
-		return aggs.GenerateEvent.GeneratedRecords, aggs.GenerateEvent.GeneratedCols, nil
-	}
```

Comment:
Why don't you need this anymore?

Reply:
I have made the code to use a similar framework as inputlookup, when hasGenerateEventWithoutSearch is present the flow would go inside the postQueryBucketcleaning and there we would populate the recs, recordIndexInFinal and finalCols via performAggOnResult method.